clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -632,6 +628,7 @@ scheduler_tmpl_t<RecordType, ReaderType>::set_initial_schedule(
                            workload_idx, input_idx, min_time,
                            inputs_[input_idx].next_timestamp);
                     inputs_[input_idx].base_timestamp = min_time;
+                    inputs_[input_idx].order_by_timestamp = true;
```

Comment:
Yes, the comment by its declaration explains that it equals the options_ value.  (There may be a way to use a lambda and pass `this` to the comparator to avoid this but I gave up on getting that to compile back in the PR that first added it.)